app/models/katello/content_view_environment.rb

```diff
@@ -75,16 +68,16 @@ def priority(content_object)
       end
     end
 
-    def self.fetch_content_view_environments(labels: [], ids: [], organization:)
+    def self.fetch_content_view_environments(candlepin_names: [], ids: [], organization:)
       # Must do maps here to ensure CVEs remain in the same order.
       # Using ActiveRecord .where will return them in a different order.
       if ids.present?
-        ids.map! do |id|
+        cves = ids.map do |id|
```

Comment:
The ! here was mutating `params[:content_view_environment_ids]` and causing the error message to be incorrect with uncompacted arrays like `No content view environments found with ids: [nil]`.